qiskit/circuit/quantumcircuit.py

```diff
@@ -2205,27 +2205,41 @@ def measure_active(self, inplace: bool = True) -> Optional["QuantumCircuit"]:
         else:
             return None
 
-    def measure_all(self, inplace: bool = True) -> Optional["QuantumCircuit"]:
+    def measure_all(
+        self, inplace: bool = True, add_creg: bool = True
+    ) -> Optional["QuantumCircuit"]:
         """Adds measurement to all qubits. Creates a new ClassicalRegister with a
-        size equal to the number of qubits being measured.
+        size equal to the number of qubits being measured if `add_creg = True`.
```

Comment:
It's good that you updated this.  We should be a bit more descriptive about what happens if you set `add_bits=False` as well - something like

> Add a measurement to all qubits.
>
> By default, a register of new classical bits is added to the circuit to store these measurements.  If the argument ` ``add_bits=False`` `, then the results will be stored in the classical bits that are already present, with qubit ` ``n`` ` being measured into classical bit ` ``n`` `.

(as an aside: in Sphinx, if you just want monospaced text, you need to use two backticks (` ``monospaced`` `), not just one)